apps/spruce/src/pages/host/Metadata.tsx

```diff
@@ -18,7 +18,7 @@ const { gray } = palette;
 export const Metadata: React.FC<{
   loading: boolean;
   host: HostQuery["host"];
-  error: ApolloError;
+  error?: ApolloError;
```

Comment:
nit: this is more aligned with actual usage
```suggestion
  error: ApolloError | undefined;
```